dist/API_DOCUMENTATION.md

```diff
@@ -206,24 +231,30 @@ No public properties.
 
 ### Methods
 - `createTransaction(recipientAddr, value, fee, nonce)`
+- `dump()`
 
 ### Events
 No events.
 
 ### Examples
 Create a transaction:
-```
+```js
 $.wallet.createTransaction(recipientAddr, value, fee, nonce).then(transaction => {
     console.log(transaction)
 })
 ```
 
+Dump and restore wallet:  
+```js
+var walletSeed = $.wallet.dump();
```

Comment:
I named the function `dump` because in theory it's not really a seed but only the private key.
Maybe rename all reference to a seed ? (like the variable name also..)